src/main/scala/subsystem/RocketSubsystem.scala

```diff
@@ -50,12 +51,19 @@ trait HasRocketTiles extends HasTiles
     rocket
   }
 
+  rocketTiles.map(r => LogicalModuleTree.add(logicalTree, r.rocketLogicalTree))
+
   def coreMonitorBundles = (rocketTiles map { t =>
     t.module.core.rocketImpl.coreMonitorBundle
   }).toList
 
+  /**
+    * getOMRocketInterruptTargets and getOMRocketCores are deprecated
+    *
+    * @return
+    */
   def getOMRocketInterruptTargets(): Seq[OMInterruptTarget] =
-    rocketTiles.flatMap(c => c.cpuDevice.getInterruptTargets())
+    rocketTiles.flatMap(c => c.rocketLogicalTree.getInterruptTargets())
 
   def getOMRocketCores(resourceBindingsMap: ResourceBindingsMap): Seq[OMComponent] =
     rocketTiles.flatMap(c => c.cpuDevice.getOMComponents(resourceBindingsMap))
```

Comment:
I still don't understand why `getOMRocketCores` is using `cpuDevice`, and `getOMRocketInterruptTargets` is using `getOMRocketInterruptTargets`.

I think you should go ahead and remove all the deprecated functions now. If you need to compare the outputs to the old one, you can just generate the file on `master` and compare that way.

Reply:
I will remove all of the deprecated functions.

Reply:
getOMRocketCores was removed.

The  deprecated functions should have been removed.

Reply:
WRT to the question about "getOMRocketInterruptTargets is using getOMRocketInterruptTargets." we should talk about the following:

```
  val ebssLogicalTree = new EBSSLogicalTreeNode(getOMCabooseInterruptTargets, getOMRocketInterruptTargets, getOMInterruptDevice)
  LogicalModuleTree.add(ebssLogicalTree, logicalTree)
```

```
 def getOMCabooseInterruptTargets(): Seq[OMInterruptTarget] =
    cabooseTiles.flatMap(c => c.cabooseLogicalTree.getOMInterruptTargets())
  cabooseTiles.map(c => LogicalModuleTree.add(logicalTree, c.cabooseLogicalTree))
```

```
  rocketTiles.map(r => LogicalModuleTree.add(logicalTree, r.rocketLogicalTree))
```